ios/engine/KMEI/KeymanEngine/Classes/Keyboard/KeymanWebViewController.swift

```diff
@@ -744,6 +742,8 @@ extension KeymanWebViewController: KeymanWebDelegate {
     // Reset the keyboard's size.
     keyboardSize = kbSize
 
+    setBannerHeight(to: Int(InputViewController.topBarHeight))
```

Comment:
This call has been relocated; it will now be triggered consistently upon the host page's load.  We never turn the banner off, and we keep the image banner and suggestion banner at the same height.

The image banner's distortion occurred because the original height-setting location... was only ever called when a lexical model was activated.  That doesn't happen for image banners.